tests/js/server/singleTemplateRouter-tests.js

```diff
@@ -0,0 +1,176 @@
+// Tests for the "single template router".
+//
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("../../../index");
+
+var jqUnit = require("node-jqunit");
+var fs     = require("fs");
+var path   = require("path");
+var jsdom  = require("jsdom");
+
+var jqueryPath = path.resolve(__dirname, "../../../node_modules/infusion/src/lib/jquery/core/js/jquery.js");
```

Comment:
This remains one of the bad forms of reference - "%infusion/src/lib/jquery/core/js/jquery.js" should work fine using infusion modules